hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieMultiTableDeltaStreamer.java

```diff
@@ -213,6 +213,26 @@ public void testMultiTableExecutionWithParquetSource() throws IOException {
     }
   }
 
+  @Test
+  public void testTableLevelProperties() throws IOException {
+    HoodieMultiTableDeltaStreamer.Config cfg = TestHelpers.getConfig(PROPS_FILENAME_TEST_SOURCE1, dfsBasePath + "/config", TestDataSource.class.getName(), false);
+    HoodieMultiTableDeltaStreamer streamer = new HoodieMultiTableDeltaStreamer(cfg, jsc);
+    List<TableExecutionContext> tableExecutionContexts = streamer.getTableExecutionContexts();
+    tableExecutionContexts.forEach(tableExecutionContext -> {
+      switch (tableExecutionContext.getTableName()) {
+        case "dummy_table_uber":
+          String publicGeneratorClass = tableExecutionContext.getProperties().getString(DataSourceWriteOptions.KEYGENERATOR_CLASS_OPT_KEY());
```

Comment:
Let us rename the variable to defaultKeyGeneratorClass?

Reply:
nit: Rather than having a case statement for `dummy_table_uber`, we can move this logic to default which is currently empty since you are only overriding for `dummy_table_short_trip` table. :)

Reply:
@pratyakshsharma  if onle hava a case, we can use if(xxxx){}else(){},what do you think so？

Reply:
@pratyakshsharma It's ok